requirements.txt

```diff
@@ -1 +1 @@
-pyparsing==2.0.2

```

Comment:
Yeah, that sounds right. I'll also look into this at some point:
https://caremad.io/2013/07/setup-vs-requirement/